net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,181 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=http://www.yo2loj.ro/hamprojects
+PKG_MD5SUM:=5429401ab6f8f793448bef11029d7ead
```

Comment:
This looks like to me as md5, which was deprecated here. You should switch to PKG_HASH, which uses sha256